js/lesson-24/part1.js

```diff
@@ -0,0 +1,39 @@
+`use strict`;
+
+// введення даних та валідація
+let $startNumber;
+let $endNumber;
+
+$startNumber = +prompt(`Введіть число - початок діапазону`);
+
+if ($startNumber === `` || isNaN($startNumber)) {
+    console.log(`Не прийнято! Спробуйте ще раз.`);
+} else {
+    console.log(`Діапазон починається з`, $startNumber);
+    $endNumber = +prompt(`Введіть число - закінчення діапазону`);
+
+    if ($endNumber === `` || isNaN($endNumber)) {
+        console.log(`Не прийнято! Спробуйте ще раз.`);
+    } else {
+        console.log(`Діапазон закінчується на`, $endNumber);
+    }
+}
+
+// розрахунок
+
+let $sum = 0;
```

Comment:
You need to `$sum = $startNumber` here, since inside your loop is `$sum++`

Or change loop (as per comment below)

Reply:
Fixed